packages/assistify-help-request/client/views/tabbar/HelpRequestActions.js

```diff
@@ -95,33 +95,26 @@ Template.HelpRequestActions.events({
 			closeOnConfirm: false,
 			roomId: instance.data.roomId
 		}), (inputValue) => {
-			/*if (!inputValue) {
-			 swal.showInputError(t('Please_add_a_comment_to_close_the_room'));
-			 return false;
-			 }
+			//inputValue is false on "cancel" and has a string value of the input if confirmed.
+			if (!(typeof inputValue === 'boolean' && inputValue === false)) {
```

Comment:
@ruKurz I could have also validated that there is a String provided for the same result.
However, as an empty String (`""`) is evaluated as false in JS, I opted for the inverse.-
wdyt?

Reply:
As non JS expert I would say:

```
if (isNotEmptyOrWhitespaceOnly(inputValue)) {
  // close
} else {
  // cancel
}
```

Reply:
I think there should be three cases handled when closing a conversation:

1. Close and store
2. Close and do not store
3. Do not close / cancel the dialog

As far as I understand the code, now it is possible to 
1. close and store
2. cancel the dialog

@mrsimpson WDYT?

Reply:
@ruKurz what's the business value of "Close and do not store"
Would this be an explicit decision by the user?

Reply:
Yes, I think the "explicit decision" is a benefit. But if you aim to grab as much data with Smarti as possible the "close and do not sore option" would cause the lose of conversational data.

So the trade-off is "user control" vs. "data octopus".

Reply:
@ruKurz I extracted the check into a separate method. Any objections merging it?

Reply:
@ruKurz whether we provide a "close but don't store" is another decision, imho.